src/main/java/eu/yals/Endpoint.java

```diff
@@ -13,45 +10,148 @@ private Endpoint() {
         throw new UnsupportedOperationException("Utility class");
     }
 
-    @RequestMethod(value = HttpMethod.POST, api = true)
-    public static final String STORE_API = "/api/store";
+    /**
+     * Endpoints accessible via browser
+     */
+    public static class UI {
+        /**
+         * Application home page
+         */
+        public static final String HOME_PAGE = "";
+
+        /**
+         * Page that will appear, when user hits location which is not exist nor served by application
+         */
+        public static final String PAGE_404 = "errors/404";
+
+        /**
+         * Page that will shown when application hits server-side error
+         */
+        public static final String ERROR_PAGE_500 = "errors/500";
+
+        /**
+         * Page that will shown when application hits fatal error and became unaccessible
+         */
+        public static final String ERROR_PAGE_503 = "errors/503";
+
+        /**
+         * Page for debugging staff (avoid adding it in PROD mode)
+         */
+        public static final String DEBUG_PAGE = "debug";
+
+        /**
+         * Page for displaying software info (avoid adding it in PROD mode)
+         */
+        public static final String APP_INFO_PAGE = "appInfo";
+    }
+
+    /**
+     * API Endpoints
+     */
+    public static class Api {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Api' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)